tools/percolator/test-data/decoy.mzid

```diff
@@ -0,0 +1,761 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MzIdentML id="MS-GF+" version="1.1.0" xmlns="http://psidev.info/psi/pi/mzIdentML/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://psidev.info/psi/pi/mzIdentML/1.1 http://www.psidev.info/files/mzIdentML1.1.0.xsd" creationDate="2016-04-08T12:44:47" >
```

Comment:
Your test data is quite large. Try to use `compare="contains"` for output files.